src/solver/opamActionGraph.ml

```diff
@@ -30,19 +30,27 @@ let name_of_action = function
   | `Reinstall _ -> "recompile"
   | `Build _ -> "build"
 
-let symbol_of_action = function
+let symbol_of_action =
+  let open OpamConsole in
+  function
   | `Remove _ ->
-      OpamConsole.utf8_symbol OpamConsole.Symbols.circled_division_slash ""
+      utf8_symbol Symbols.circled_division_slash
+                  ~alternates:[Symbols.greek_small_letter_lambda] "X"
```

Comment:
We already have a fallback to more explicit strings (just above) in case UTF8 is not available. Are the single-letter fallbacks really needed ?
I figure the intent is to allow partial fallback if only some of the symbols are not available ?

Reply:
Indeed - the alternates list is based on looking at the console fonts in previous versions of Windows and then having the single letter for complete failure. FWIW it was only for completeness - Consolas has something useful on all support versions of Windows.